src/helper/helper.js

```diff
@@ -161,8 +172,14 @@ class DataLayerHelper {
    * this method has been executed. Unless the processNow argument is
    * intentionally set to false via the constructor, this method will
    * always execute at construction time.
+   *
+   * Note: This method should only be called a single time to prepare
+   * the helper.
+   * @export
    */
   process() {
+    logError(`Process has already been ran. This method should only ` +
+        `run a single time to prepare the helper.`, LogLevel.ERROR);
```

Comment:
It looks like this message appears every time rather than just the first time?

Reply:
Whoops, that was a complete brain fart, fixed 😆

Reply:
It might be safer to set a boolean variable, since somebody could register a processor that does something else on set, then call process for the first time correctly